### PR TITLE
test changes

### PR DESCRIPTION
**Pull request by thecraftman**
test oout changes

### CODE REVIEW

File: docs/reference/api.md [hunk 1]
```diff
@@ -1,5 +1,255 @@
...
\ No newline at end of file
+The best way to get started with our API is to use [Go-Axiom](https://github.com/axiomhq/axiom-go) to send events directly to Axiom API.
+
+Axiom understands your resources and provides an API to ingest structured data logs, handle and manage your deployments. This is a **REST-style** API that uses JSON for serialization and OAuth 2 for authentication. 
+
+This page covers the basics for interacting with the Axiom API, plus instructions for ingesting data and notes on some commonly used endpoints.
+
+## Authentication
+
+Axiom uses OAuth2 for authentication. All requests must use HTTPS.
```
Comment (bahlo) [inline docs/reference/api.md]: We don't use OAuth (2)

File: docs/reference/api.md [hunk 2]
```diff
@@ -1,5 +1,255 @@
...
+You can generate a ingest token in your Axiom user settings for manual authentication in shell scripts or commands that use the Axiom API. Refer to the code sample below for an example of how to use this token in a curl request.
+
+### API Tokens
+
+Axiom has two types of API tokens; 
+
+- Personal API Tokens.
+- Ingest Tokens.
+
+### Personal API Token;
```
Comment (bahlo) [inline docs/reference/api.md]:
```suggestion
### Personal API Token
```

File: docs/reference/api.md [hunk 3]
```diff
@@ -1,5 +1,255 @@
...
+- Personal API Tokens.
+- Ingest Tokens.
+
+### Personal API Token; 
+
+The personal token can be retrieved from the users profile page, all users have a Personal Token. With the Personal Token users can access the Axiom API programmatically for custom integrations, management setting, or for tools such as the Axiom CLI.
+
+The personal access token grants access to all resources available to the user on his behalf.  
+
+Personal Access or Ingest token. Can be created under **Profile** or **Settings > Profile > Personal Tokens.**
```
Comment (bahlo) [inline docs/reference/api.md]:
```suggestion
A personal token can be created under **Profile** or **Settings > Profile > Personal Tokens.**
```

File: docs/reference/api.md [hunk 4]
```diff
@@ -1,5 +1,255 @@
...
+You can obtain the Ingest Token from the settings > Ingest Token of the Axiom deployment.
+
+<img class="axi-window-shadow" src="/assets/shots/ingest-token.png" alt="Ingest Token overview" /> 
+
+
+Also, we have two API clients for your convenience:
+
+- Go Client 
+
+- JS Client
```
Comment (bahlo) [inline docs/reference/api.md]: Can we link them?

File: docs/reference/api.md [hunk 5]
```diff
@@ -1,5 +1,255 @@
...
+
+Individual events are ingested as an HTTP POST request.
+
+
+### Using curl command to ingest data 
+
+
+```
+curl -X POST '$YOUR_AXIOM_URL/api/v1/ingest' \
+  -H 'Authorization: Bearer $INGEST_TOKEN' \
```
Comment (bahlo) [inline docs/reference/api.md]: or `$PERSONAL_TOKEN`

File: docs/reference/api.md [hunk 6]
```diff
@@ -1,5 +1,255 @@
...
+  -H 'Content-Type: application/x-ndjson' -H 'x-axiom-org-id: axiom' \
+  -d '{"container": {"image": "front:master.0326.5","name": "tunnel-front"},"labels": {"component": "tunnel","pod-template-hash": "75f5666499"},"namespace": "kube-system","node": {"name": "32068356-vmss000002"},"pod": {"name": "fnxln","uid": "f0bdea4e57"},"replicaset": {"name": "75f5666499"}}}'
+
+```
+
+### Example Response
+
+```
+
+A successful POST Request returns a standard HTTP 200 response code.
```
Comment (bahlo) [inline docs/reference/api.md]: We actually return 200 and JSON with details

**Comment by thecraftman** [comment]
- API section
- Broken links
- Redirect

**Pull request merged**